qa/common/gen_qa_custom_ops

```diff
@@ -56,11 +57,12 @@ cp ./gen_qa_custom_ops_models.py $HOST_SRCDIR/.
 cp ./cuda_op_kernel.cu.cc.patch $HOST_SRCDIR/.
 cp ./busy_op_kernel.* $HOST_SRCDIR/.
 TFSCRIPT=tf_gen.cmds
+C2SCRIPT=c2_gen.cmds
```

Comment:
Shouldn't it be PYTSCRIPT=pyt_gen.cmds or similar?

Reply:
Done